apps/coscrad-frontend/src/components/higher-order-components/selected-categorizables-of-multiple-types.presenter.tsx

```diff
@@ -0,0 +1,75 @@
+import {
+    CategorizableType,
+    IBaseViewModel,
+    ICategorizableDetailQueryResult,
+} from '@coscrad/api-interfaces';
+import { ErrorDisplay } from '../error-display/error-display';
+import { Loading } from '../loading';
+import { ICategorizableDetailPresenterFactory } from '../resources/factories/categorizable-detail-presenter-factory.interface';
+import { SelectedCategorizablesPresenter } from './selected-categorizables-of-single-type.presenter';
+import {
+    getCategorizableTypeForSliceKey,
+    ViewModelDetailSnapshot,
+    ViewModelIndexSnapshot,
+} from './use-loadable-categorizables';
+
+import { isNonEmptyString, isNullOrUndefined } from '@coscrad/validation-constraints';
+
+interface SelectedCategorizablesOfMultipleTypesPresenterProps<
+    T extends IBaseViewModel = IBaseViewModel
+> {
+    viewModelSnapshot: ViewModelDetailSnapshot;
+    presenterFactory: ICategorizableDetailPresenterFactory<ICategorizableDetailQueryResult<T>>;
+    getPluralLabelForCategorizableType: (categorizableType: CategorizableType) => string;
+    heading?: string;
+}
+
+export const SelectedCategorizablesOfMultipleTypesPresenter = <T extends IBaseViewModel>({
+    viewModelSnapshot,
+    presenterFactory,
+    getPluralLabelForCategorizableType,
+    heading,
+}: SelectedCategorizablesOfMultipleTypesPresenterProps): JSX.Element => {
+    return (
+        <>
+            <h2>{isNonEmptyString(heading) ? heading : 'Connected Resources'}</h2>
```

Comment:
Might be nice to have an abstraction around defaults. Seems like this should be the responsibility of the content config itself.